EveripediaNetwork/ep-ui
# Improve the Mobile Menu Orientation

# Improve the Mobile Menu Orientation

This PR:
- Allows the mobile menu to take maximum height on mobile 
- Adds Chinese to the list of languages

## How should this be tested?

1. Resize the screen to view the homepage on mobile
2. Click on the list of languages to see newly added langauage


## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2451
